ena-submission/src/ena_deposition/create_sample.py

```diff
@@ -182,13 +187,13 @@ def set_sample_table_entry(db_config, row, seq_key):
         }
         update_db_where_conditions(
             db_config,
-            table_name="submission_table",
+            table_name=TableName.SUBMISSION_TABLE,
             conditions=seq_key,
             update_values=update_values,
         )
 
 
-def submission_table_start(db_config: SimpleConnectionPool):
+def submission_table_start(db_config: SimpleConnectionPool, config: Config):
```

Comment:
I don't think we need config here?

Reply:
I wouldn't have added it if it wasn't
<img width="708" alt="image" src="https://github.com/user-attachments/assets/e9482a96-ebb8-42a0-927f-8132f8c09431" />